c/doc/Linux-Almach-C.md

```diff
@@ -0,0 +1,171 @@
+Run a simple C sample on Almach device running Yocto Linux
+===
+---
+
+# Table of Contents
+
+-   [Introduction](#Introduction)
+-   [Step 1: Prerequisites](#Prerequisites)
+-   [Step 2: Prepare your Device](#PrepareAlmach)
+-   [Step 3: Build and Run the Sample](#Build)
+-   [Tips](#tips)
+
+<a name="Introduction"></a>
+# Introduction
+
+**About this document**
+
+This document describes how to connect Almach device running Yocto Linux with Azure IoT SDK. This multi-step process includes:
+-   Configuring Azure IoT Hub
+-   Registering your IoT device
+-   Build and deploy Azure IoT SDK on device
+
+<a name="Prerequisites"></a>
+# Step 1: Prerequisites
+
+You should have the following items ready before beginning the process:
+
+-   Computer with Git client installed and access to the
+    [azure-iot-sdks](https://github.com/Azure/azure-iot-sdks) GitHub
+    public repository.
+-   Almach device
+-   For wired connection, [USB Ethernet board](http://www.e-consystems.com/DM3730-development-board.asp#accessories)
+-   PC to build Yocto Image
+-   Terminal Emulator software
+-   Download and install [DeviceExplorer](https://github.com/Azure/azure-iot-sdks/releases/download/2015-11-13/SetupDeviceExplorer.msi).
+-   [Set up your IoT hub](https://github.com/Azure/azure-iot-sdks/blob/master/doc/setup_iothub.md).
+
+#### Create a device on IoT Hub
+-   With your IoT hub configured and running in Azure, follow the instructions in **"Create Device"** section of [DeviceExplorer Usage document](https://github.com/Azure/azure-iot-sdks/blob/master/tools/DeviceExplorer/doc/how_to_use_device_explorer.md).
+
+#### Write down device credentials
+-   Make note of the Connection String for your device by following the instructions in **"Get device connection string or configuration data"** section of [DeviceExplorer Usage document](https://github.com/Azure/azure-iot-sdks/blob/master/tools/DeviceExplorer/doc/how_to_use_device_explorer.md).
+
+<a name="PrepareAlmach"></a>
+# Step 2: Prepare Almach
+-   Prepare the Yocto Root file system for your Almach board by following the instructions in the "esom3730_yocto_user_manual.pdf".
```

Comment:
Is the file esom3730_yocto_user_manual.pdf available at some URL? if yes, could you please enter the URL where user can find the file?